packages/use-dataloader/src/DataLoaderProvider.tsx

```diff
@@ -24,6 +24,7 @@ type UseDataLoaderInitializerArgs<T = unknown> = {
   method: () => PromiseType<T>
   pollingInterval?: number
   keepPreviousData?: boolean
+  maxDataLifetime?: number
```

Comment:
Can we add it into documentation with a precision that this value is in milliseconds ?